app/views/layouts/base.html.erb

```diff
@@ -33,11 +33,31 @@
   <![endif]-->
   <%= csrf_meta_tags %>
 
-  <link href="/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180" />
-  <link href="/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png" />
-  <link href="/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png" />
+  <%= tag.link(
+        href: asset_path(design_system_asset_path('img/favicons/apple-touch-icon.png')),
+        rel: 'apple-touch-icon',
+        sizes: '180x180',
+        type: 'image/png',
+      ) %>
+  <%= tag.link(
+        href: asset_path(design_system_asset_path('img/favicons/favicon-32.png')),
```

Comment:
This file doesn't exist. Maybe we can use the `40x40` one instead?

Reply:
I just saw that, should be fixed in the last commit